include/rocksdb/options.h

```diff
@@ -1823,6 +1834,21 @@ struct CompactRangeOptions {
   // Cancellation can be delayed waiting on automatic compactions when used
   // together with `exclusive_manual_compaction == true`.
   std::atomic<bool>* canceled = nullptr;
+
+  // If set to kForce, it will perform garbage collection (GC) of blobs that are
+  // no longer referenced by any live SST files. If set to kDisable, it will not
+  // perform GC. Both kForce and kDisable will override user-provided setting in
+  // DBOptions.enable_blob_garbage_collection. This enables customers to both
+  // force-enable and force-disable GC when calling CompactRange as well as
+  // selectively override the age cutoff.
```

Comment:
This description of what blob GC does is not entirely accurate: during GC, *valid* blobs in files below the cutoff are relocated; blob files are then dropped as a unit once they are all garbage/not referenced anymore.

Anyways, I think it would be sufficient here to explain that `kForce` overrides `enable_blob_file_garbage_collection` to true, `kDisable` overrides it to false, and `kUseDefault` leaves the setting in effect.

Reply:
done